docs/source/geometry.transform.rst

```diff
@@ -9,6 +9,7 @@ The functions in this section perform various geometrical transformations of 2D
 .. autofunction:: warp_affine
 .. autofunction:: get_perspective_transform
 .. autofunction:: get_rotation_matrix2d
+.. autofunction:: normal_transform_pixel
```

Comment:
@ucalyptus thanks for the contribution. Have to apologize because it was not specified enough in the issue but this function needs also typing, and some description. Take a look at others: https://github.com/kornia/kornia/blob/master/kornia/geometry/transform/imgwarp.py#L273

Actually just realizing that many functionalities from this source file need yet to be typed. Can be done later ;D